File: Example_Clients/CameraClient/MicrosoftCognitiveServiceExample/Configuration.cs
using System;
using System.IO;

namespace MicrosoftCognitiveServiceExample
{
    public class Configuration
    {   

        public static void ReadConfig(String Filename)
        {
            // Read every line in the file.
            using (StreamReader reader = new StreamReader(Filename))
            {
                string line;
                while ((line = reader.ReadLine()) != null)
                {
                    // Split line using before equals as key word and after equals as value
                    string[] parts = line.Split('=');

                    if (parts != null && parts.Length == 2 && parts[0].StartsWith("SubscriptionKeyForEmotion"))
                    {
                        UserInterface.SUBSCRIPTION_KEY_FOR_EMOTION = parts[1];                       
                    }
                    if (parts != null && parts.Length == 2 && parts[0].StartsWith("DefaultCameraIP"))
                    {
                        UserInterface.DEFAULT_IP = parts[1];
                    }
                    if (parts != null && parts.Length == 2 && parts[0].StartsWith("PicturePath"))
                    {
                        UserInterface.PICTURE_PATH = parts[1];
                    }
                }
            }
        }
    }
}

File: Example_Clients/CameraClient/CognitiveServiceExample/src/Configuration.java
import java.io.*;

/**
 * Configuration class load the default camera IP, subscription key and temporary picture path from the cfg file
 */

public class Configuration {

    public static void ReadConfig(String filename) {

        try {
            //  Read every line in the file.

            BufferedReader br = new BufferedReader(new FileReader(filename));
            String line;
            while ((line = br.readLine()) != null) {
                //  Split line using before equals as key word and after equals as value
                String[] parts = line.split("=");
                if (((parts != null) && ((parts.length == 2) && parts[0].startsWith("SubscriptionKeyForEmotion")))) {
                    UserInterface.SUBSCRIPTION_KEY_FOR_EMOTION = parts[1];
                }

               if (((parts != null) && ((parts.length == 2) && parts[0].startsWith("DefaultCameraIP")))) {
                    UserInterface.DEFAULT_IP = parts[1];
                }

                if (((parts != null)  && ((parts.length == 2) && parts[0].startsWith("PicturePath")))) {
                    UserInterface.PICTURE_PATH = parts[1];
                }

            }
        }
        catch (Exception ex)
        {
            ex.printStackTrace();
        }

    }
}


